store/src/main/java/org/apache/rocketmq/store/DefaultMessageStore.java

```diff
@@ -782,16 +783,28 @@ private String getStorePathPhysic() {
     public HashMap<String, String> getRuntimeInfo() {
         HashMap<String, String> result = this.storeStatsService.getRuntimeInfo();
 
-        {
-            double physicRatio = UtilAll.getDiskPartitionSpaceUsedPercent(getStorePathPhysic());
+        String commitLogStorePath = DefaultMessageStore.this.getMessageStoreConfig().getStorePathCommitLog();
```

Comment:
Dose it support dledge mode? the original code getStorePathPhysic() has different value of  commitLogStorePath in dledge mode

Reply:
Thank you for the reminding. 
Changed to `getStorePathPhysic()`